### ISSUE TITLE
device class None, state class None and unit thus indicating it has a numeric value; 

### ISSUE THREAD

**Issue by Next9999** [sfstar/hass-victron#63]
Logger: homeassistant.components.sensor
Source: components/sensor/__init__.py:597
Integration: Sensor (documentation, issues)
First occurred: 08:16:39 (77 occurrences)
Last logged: 08:16:40

Sensor sensor.victron_inverter_alarm_lowacoutvoltage_239 has device class None, state class None and unit thus indicating it has a numeric value; however, it has the non-numeric value: OK (<class 'str'>); Please update your configuration if your entity is manually configured, otherwise report it to the custom integration author.
Sensor sensor.victron_inverter_alarm_overload_239 has device class None, state class None and unit thus indicating it has a numeric value; however, it has the non-numeric value: OK (<class 'str'>); Please update your configuration if your entity is manually configured, otherwise report it to the custom integration author.
Sensor sensor.victron_inverter_alarm_ripple_239 has device class None, state class None and unit thus indicating it has a numeric value; however, it has the non-numeric value: OK (<class 'str'>); Please update your configuration if your entity is manually configured, otherwise report it to the custom integration author.
Sensor sensor.victron_inverter_info_state_239 has device class None, state class None and unit thus indicating it has a numeric value; however, it has the non-numeric value: INVERTING (<class 'str'>); Please update your configuration if your entity is manually configured, otherwise report it to the custom integration author.
Sensor sensor.victron_inverter_alarm_lowsoc_239 has device class None, state class None and unit thus indicating it has a numeric value; however, it has the non-numeric value: OK (<class 'str'>); Please update your configuration if your entity is manually configured, otherwise report it to the custom integration author.

**Comment by ISO8807** [comment]
I have Home Assistant 2023.3.5, Supervisor 2023.03.1, Operating System 9.5, Frontend 20230309.1. I too am getting  similar log entries when communicating with my Cerbo GX:

```
WARNING (MainThread) [homeassistant.components.sensor] Sensor
sensor.victron_dcsystem_alarm_highvoltage_226 has device class None, state class
None and unit  thus indicating it has a numeric value; however, it has the
non-numeric value: OK (<class 'str'>); Please update your configuration if your
entity is manually configured, otherwise report it to the custom integration
author.

WARNING (MainThread) [homeassistant.components.sensor] Sensor
sensor.victron_dcsystem_alarm_lowstartervoltage_226 has device class None, state
class None and unit  thus indicating it has a numeric value; however, it has the
non-numeric value: OK (<class 'str'>); Please update your configuration if your
entity is manually configured, otherwise report it to the custom integration
author.

WARNING (MainThread) [homeassistant.components.sensor] Sensor
sensor.victron_dcsystem_alarm_highstartervoltage_226 has device class None,
state class None and unit  thus indicating it has a numeric value; however, it
has the non-numeric value: OK (<class 'str'>); Please update your configuration
if your entity is manually configured, otherwise report it to the custom
integration author.

WARNING (MainThread) [homeassistant.components.sensor] Sensor
sensor.victron_dcsystem_alarm_lowtemperature_226 has device class None, state
class None and unit  thus indicating it has a numeric value; however, it has the
non-numeric value: OK (<class 'str'>); Please update your configuration if your
entity is manually configured, otherwise report it to the custom integration
author.

WARNING (MainThread) [homeassistant.components.sensor] Sensor
sensor.victron_dcsystem_alarm_hightemperature_226 has device class None, state
class None and unit  thus indicating it has a numeric value; however, it has the
non-numeric value: OK (<class 'str'>); Please update your configuration if your
entity is manually configured, otherwise report it to the custom integration
author.
```

**Comment by sfstar** [comment]
Hello and thank you for opening this issue.
It has been resolved in version v0.0.10 and the issue is primarily tracked in the thread #69.
Feel free to re-open if upgraden to v0.0.10 doesn't solve your issue
